src/NuGet.Clients/NuGet.PackageManagement.UI/UserInterfaceService/INuGetUI.cs

```diff
@@ -80,9 +79,14 @@ public interface INuGetUI : IDisposable
         bool DisplayDeprecatedFrameworkWindow { get; }
 
         /// <summary>
-        /// Package currently selected in the UI
+        /// Packages currently selected in the UI in the packages list
         /// </summary>
-        PackageIdentity SelectedPackage { get; }
+        IEnumerable<PackageItemViewModel> SelectedPackages { get; }
```

Comment:
It always have the selected package(s) even if it is just one for install/uninstall.

Reply:
Consider this scenario. I believe the Selected items is 0. Notice there's no details because of this.
However, "selected for Update" is 4. What do you think?

![image](https://user-images.githubusercontent.com/49205731/135004873-cec0814d-f1c5-48f9-8ca6-ff1d10e53bcd.png)